src/ForgotPasswordController.js

```diff
@@ -74,17 +78,25 @@ function (Okta, FormController, Enums, FormType, ValidationUtil, ContactSupport,
           });
         })
         .fail(function () {
-          self.set('factorType', Enums.RECOVERY_FACTOR_TYPE_EMAIL);
         });
       }
     },
     Form: {
-      autoSave: true,
-      save: _.partial(Okta.loc, 'password.forgot.sendEmail', 'login'),
+      noButtonBar: true,
       title: _.partial(Okta.loc, 'password.reset', 'login'),
       formChildren: function () {
-        var formChildren = [
-          FormType.Input({
+        /*eslint complexity: [2, 9] */
+        var smsEnabled = this.settings.get('features.smsRecovery');
+        var callEnabled = this.settings.get('features.callRecovery');
+        var emailEnabled = this.settings.get('features.emailRecovery');
+        var noFactorsEnabled = !(smsEnabled || callEnabled || emailEnabled);
+        var formChildren = [];
+
+        if(noFactorsEnabled) {
```

Comment:
I think in general we prefer a space in between ```if``` and ```(```, so if you can change them to ```if (``` that will be great